utils.mk

```diff
@@ -3,6 +3,27 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
+# Owner for installed files
+INSTALL_OWNER     ?= root
+
+# Group for installed files
+INSTALL_GROUP     ?= adm
```

Comment:
https://refspecs.linuxfoundation.org/LSB_5.0.0/LSB-Core-generic/LSB-Core-generic/usernames.html

It's strictly optional but afaik it is provided by the main distros. But if not, those distros could run `make INSTALL_GROUP=root` if they wished.

Reply:
I think the `adm` group is present in the majority of the distros, @liubin.

Not sure about darwin, though. /cc @sameo @egernst

Reply:
@why not just keep the group as root? Whats the benefit of having admin here?

Reply:
We could make it `root` but I chose `adm` to give admins a change to add select users to that group (rather than the `root` group) in recognition that this PR _may_ cause issues for some as currently any user can run the binaries, for instance.